src/settings-ui/Microsoft.PowerToys.Settings.UI/Views/ImageResizerPage.xaml

```diff
@@ -179,12 +179,11 @@
 
                     <controls:Setting x:Uid="ImageResizer_Encoding">
```

Comment:
Possibly off-topic, but I notice the above control (on line 165) has `x:Uid="Encoding"`. Shouldnt this one be named something with "compression" or "quality" in it?

Reply:
Out of scope of this PR